flow/layers/opacity_layer.cc

```diff
@@ -56,18 +56,15 @@ void OpacityLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
   Layer::AutoPrerollSaveLayerState save =
       Layer::AutoPrerollSaveLayerState::Create(context);
 
-  // Collect inheritance information on our children in Preroll so that
-  // we can decide whether or not to use a saveLayer in Paint.
-  context->subtree_can_inherit_opacity = true;
-  // ContainerLayer will turn the flag off if any children are
-  // incompatible or if they overlap
   ContainerLayer::Preroll(context, child_matrix);
   // We store the inheritance ability of our children for |Paint|
-  set_children_can_accept_opacity(context->subtree_can_inherit_opacity);
+  set_children_can_accept_opacity((context->renderable_state_flags &
```

Comment:
Do we use this for anything besides calling `ShouldNotBeCached` below?

Reply:
A bunch of unit tests use it to make sure that the right decision was reached.